variant_qc/exomes_genomes_coverage.py

```diff
@@ -0,0 +1,145 @@
+from gnomad_hail import *
+from gnomad_hail.resources.variant_qc import get_ucsc_mappability
+
+def import_cds_from_gtf() -> hl.Table:
+    """
+    Creates a HT with a row for each base / gene that is in a CDS in gencode v19
+    :return: HT
+    :rtype: Table
+    """
+    gtf = hl.experimental.import_gtf(
+        'gs://hail-common/references/gencode/gencode.v19.annotation.gtf.bgz',
+        reference_genome='GRCh37',
+        skip_invalid_contigs=True, min_partitions=200
+    )
+    gtf = gtf.filter((gtf.feature == 'CDS') & (gtf.transcript_type == 'protein_coding') & (gtf.tag == 'basic'))
+    gtf = gtf.annotate(locus=hl.range(gtf.interval.start.position, gtf.interval.end.position).map(lambda x: hl.locus(gtf.interval.start.contig, x, 'GRCh37')))
+    gtf = gtf.key_by().select('gene_id', 'locus', 'gene_name').explode('locus')
+    gtf = gtf.key_by('locus', 'gene_id').distinct()
+    return gtf.checkpoint('gs://gnomad-tmp/gencode_grch37.gene_by_base.ht', overwrite=True)
```

Comment:
consider parameterizing `overwrite`, and doing `_read_if_exists = not overwrite`